docs/components/FSXABaseComponent.md

```diff
@@ -0,0 +1,128 @@
+[Readme](../../README.md) » [BaseComponents](../README.md) » FSXABaseComponent
+
+# FSXABaseComponent
+
+1. [Introduction](#introduction)
+2. [Usage](#usage)
+3. [Properties](#properties)
+4. [Methods](#methods)
+
+## Introduction
+
+The FSXABaseComponent is there to have a basic component to be able to implement further components.
+Thereby different attributes and methods are given, which should simplify the development significantly. These are described in this section.
+
+## Usage
+
+To use this base component a new class has to be created which extends the `FSXABaseComponent`.
+
+```tsx
+@Component
+class Component extends FSXABaseComponent {}
+```
+
+<hr>
+If you want to have type support for your props you can provide an interface and pass it as an generic like here the `Props` interface.
```

Comment:
"_a_ generic". "like here" sounds German, I'd write "like the `Props` interface in this example" instead.